compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -1189,7 +1189,7 @@ fn should_encode_type(tcx: TyCtxt<'_>, def_id: LocalDefId, def_kind: DefKind) ->
 
 fn should_encode_const(def_kind: DefKind) -> bool {
     match def_kind {
-        DefKind::Const | DefKind::AssocConst | DefKind::AnonConst => true,
+        DefKind::Const | DefKind::AssocConst | DefKind::AnonConst | DefKind::InlineConst => true,
```

Comment:
Is this change needed because of this PR or was this an existing bug in `feature(inline_const)`?

Reply:
This was an existing bug.